src/components/Buttons/ButtonPost.tsx

```diff
@@ -2,19 +2,19 @@ import React from "react";
 import styled from "styled-components";
 
 const Submit = styled.input.attrs({ type: "submit" })`
-    background-color: ${(props) => props.theme.secondaryBackgroundColor};
+    background-color: ${(props) => props.theme.primary.background[2]};
     padding: 10px;
     border-radius: 5px;
     text-decoration: none;
-    color: ${(props) => props.theme.secondaryForegroundColor};
+    color: ${(props) => props.theme.primary.foreground[2]};
     border: 0 solid black;
 
     &:hover {
         font-weight: bold;
     }
 `;
 
-interface Props {
+export interface Props {
```

Comment:
Since this is being exported, can you rename it to something other than `Props`? :)

Reply:
Refactored